src/components/map/toolbar/toolbarHelp.tsx

```diff
@@ -1,32 +1,37 @@
 import React, { Component } from 'react';
+import { observer } from 'mobx-react';
 import ToolbarTool from '~/enums/toolbarTool';
+import ToolbarStore from '~/stores/toolbarStore';
 import * as s from './toolbarHelp.scss';
 
-interface IToolbarHelpProps {
-    tool: ToolbarTool;
-}
-
-export default class ToolbarHelp extends Component<IToolbarHelpProps> {
+@observer
+export default class ToolbarHelp extends Component {
+    private renderToolbarHelpContent() {
+        const selectedTool = ToolbarStore!.selectedTool;
+        if (!selectedTool) return null;
 
-    private getToolbarHelpContent(tool: ToolbarTool) {
-        const textContainers = {
-            edit: (
+        switch (selectedTool.toolType) {
+        case ToolbarTool.AddNewRoutePath:
+            return (
+                <div>
+                    Muodostaaksesi reitin suunnan, valitse kartalta aloitus-solmu.
+                    Tämän jälkeen jatka reitin suunnan muodostamista valitsemalla seuraavia solmuja.
+                </div>
+            );
+        case ToolbarTool.EditNetworkNode:
+            return (
                 <div>
                     Muokataksesi verkon solmua tai siihen
                     liittyviä linkkejä, valitse solmu kartalta.
                 </div>
-            ),
-        };
-        switch (tool) {
-        case ToolbarTool.Edit:
-            return textContainers.edit;
+            );
         default:
             return null;
         }
     }
 
     render() {
-        const toolbarHelpContent = this.getToolbarHelpContent(this.props.tool);
+        const toolbarHelpContent = this.renderToolbarHelpContent();
```

Comment:
I was thinking about that before... This needs more refactoring than that however because of class s.toolbarHelp should be around the <div>'s

But how those texts are brought here will be fixed in #456 so maybe I just add a TODO comment there